leopard.go

```diff
@@ -18,12 +18,49 @@ func init() {
 type leoRSFF8Codec struct{}
 
 func (l leoRSFF8Codec) Encode(data [][]byte) ([][]byte, error) {
-	return leopard.Encode(data)
+	return encode(data)
+}
+
+func encode(data [][]byte) ([][]byte, error) {
+	dataLen := len(data)
+	enc, ok := encoderCache.Query(dataLen)
+	if !ok {
+		var err error
+		enc, err = reedsolomon.New(dataLen, dataLen, reedsolomon.WithLeopardGF16(true))
```

Comment:
With https://github.com/klauspost/reedsolomon/pull/209 and @klauspost's optimization work, you can replace `WithLeopardGF16` with `WithLeopardGF` to get both GF16 and GF8 (depending on shard size).